pages/awards/index.js

```diff
@@ -117,6 +120,66 @@ export default function Performances() {
     return elements;
   };
 
+  async function getAwards() {
+    try {
+      const resp = await axios({
```

Comment:
After my PR gets merged in, it will require an eventID to be specified in the request body

Reply:
Yup, will keep that in mind!